### PR TITLE
{vis}[GCCcore/13.2.0] plantri v5.4

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/2bfa0a31aff6081a79c8fe1821d766d4 for a full test report.

File: easybuild/easyconfigs/p/plantri/plantri-5.4-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+easyblock = 'MakeCp'
+
+name = 'plantri'
+version = '5.4'
+
+homepage = 'https://doc.sagemath.org/html/en/reference/spkg/plantri.html'
+description = """Plantri is a program that generates certain types of graphs that are imbedded on the sphere.
```
Comment (verdurin) [inline easybuild/easyconfigs/p/plantri/plantri-5.4-GCCcore-13.2.0.eb:7]: @PetrKralCZ if you're feeling enthusiastic you could report the misspelling of "embedded" here to the upstream developers.

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**